content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -645,8 +644,6 @@ Each feature gate is designed for enabling/disabling a specific feature:
   kubelets on Pod log requests.
 - `AnyVolumeDataSource`: Enable use of any custom resource as the `DataSource` of a
   {{< glossary_tooltip text="PVC" term_id="persistent-volume-claim" >}}.
-- `AppArmor`: Enable use of AppArmor mandatory access control for Pods running on Linux nodes.
-  See [AppArmor Tutorial](/docs/tutorials/security/apparmor/) for more details.
```

Comment:
Please do not remove this item. The feature is still there.

Reply:
Fixed in [b986c7f](https://github.com/kubernetes/website/pull/37191/commits/b986c7f1e75a04a4469276b4d1a55171fd57562f)